presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -681,3 +681,44 @@ The following properties allow tuning the :doc:`/functions/regexp`.
     to hit the limit on matches for subsequent rows as well, you want to use the
     correct algorithm from the beginning so as not to waste time and resources.
     The more rows you are processing, the larger this value should be.
+
+
+Logging Properties
+------------------
+
+Presto uses the server application framework `Airlift
+<https://github.com/airlift/airlift>`_ and therefore supports its configuration
+properties for the HTTP server and general logging.
+
+``http-server.log.max-history``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``integer``
+    * **Default value:** ``15``
+
+    The maxiumum number of log files for the HTTP server to use, before
+    log rotation replaces old content.
+
+``http-server.log.max-size``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``data size``
+    * **Default value:** ``unlimited``
+
+    The maximum file size for the log file of the HTTP server.
```

Comment:
@s2lomon  this is before compression, right?

Reply:
(`http-server.log.compression.enabled` might be worth mentioning too)

Reply:
Added compression... log files are only compressed after rotation from what I know, not while being written to, so the file size would only count for uncompressed... right?

Reply:
In either case .. I dont think we need to go into that detail anyway.